### ISSUE TITLE
Save QC image alignments from command line

### ISSUE THREAD

**Issue by marivas-MRI** [raamana/visualqc#72]
Dear VisualQC devs, 

First of all, congratulations for this very usefull toolbox!. I just would like to ask you about image alignment quality control. Is there a way to avoid opening the QC window and automatically save the snapshots of each subject using the command line?

I used the following command and also it worked but I would like to save all the snapshots.

visualqc_alignment -i1 orig.nii -i2 aligned.nii -dl 0.5 -old 

Apologies for this basic question. 

Thank you very much in advance. 

Best regards,

**Comment by raamana** [comment]
Hi, thank you. I am glad to hear you find this useful.

Nothing to apologize for at all - this is somewhat a routine request: see #37 and #70 e.g., within Freesurfer QC and other modules of VisualQC. I resisted it as I didn't see the benefit, esp. when the interactive review is so fast and works on the latest version of the data.. and also because the screenshot can become out of sync with the latest version of the data, if great care hasn't been taken to keep them in sync.

I think I will implement it and update you soon. Can you quickly let me know what kinda of modalities and datasets will you be working with?

PS: I am also curious to learn where VisualQC users are so if you don't mind, can you tell me which country and lab are you based at?

**Comment by raamana** [comment]
Hi @marivas-MRI 

I made decent progress on batch generation of screenshots today, and it's all in the branch `screenshots_batch_generation`, both for `alignment` and `defacing` modules. I will release it soon once I test it a bit more and with other modules. 

If know how to use a python library from a separate branch, you can already use it via https://github.com/raamana/visualqc/tree/screenshots_batch_generation

but if your python skills aren't advanced, just wait for the official release

**Comment by raamana** [comment]
here is the instructions if you want to give it a try: https://github.com/raamana/visualqc/issues/37#issuecomment-1474181538

**Comment by raamana** [comment]
officially released in v0.6.5 today. use the `--screenshots_only` flag with any module of VisualQC now.

**Comment by raamana** [comment]
for future reference, collages can easily be generated for an entire dataset, with [mrivis](github.com/raamana/mrivis):

```python
from mrivis import Collage, Carpet, SlicePicker
collage = Collage()

for subject_id, image in get_images():

    collage.attach(image)

    collage.save(output_path=out_dir / f'vis_{subject_id}.png')
```
